locales/en.default.schema.json

```diff
@@ -43,8 +43,42 @@
         }
       }
     },
-    "cards": {
-      "name": "Cards",
+    "product_cards": {
+      "name": "Product cards",
+      "settings": {
+        "header__badge": {
+          "content": "Badge"
+        },
+        "style": {
+          "options__1": {
+            "label": "Standard"
+          },
+          "options__2": {
+            "label": "Card"
+          },
+          "label": "Style"
+        }
+      }
+    },
+    "blog_cards": {
+      "name": "Blog cards",
+      "settings": {
+        "header__badge": {
```

Comment:
I think I can remove this 🤔 I dont see it being used anywhere but wanted to ensure before

Reply:
Yep, we should remove `header__badge`